security-and-compliance/tls-and-certificates/ingress/issuer.hbs.md

```diff
@@ -0,0 +1,268 @@
+          name: letsencrypt-production
+        server: https://acme-v02.api.letsencrypt.org/directory
+        solvers:
+          - http01:
+              ingress:
+                class: contour
+    ```
+
+    > **Important**
+    > * `Let's Encrypts` production API has [rate
+    >   limits](https://letsencrypt.org/docs/rate-limits/).
+    > * `Let's Encrypts` requires your `shared.ingress_domain` to be accessible
+    >   from the internet
+    > * Depending on your setup you will need to adjust
+    >   [.spec.acme.solvers](https://cert-manager.io/docs/configuration/acme/#solving-challenges)
+    > * Replace `.spec.acme.email` with the email which should receive notices
+    >   for certificates from LetsEncrypt
+
+  : LetsEncrypt staging
+
+    Create a `ClusterIssuer` for [Let's Encrypts](https://letsencrypt.org) staging API:
+
+    ```yaml
+    ---
+    apiVersion: cert-manager.io/v1
+    kind: ClusterIssuer
+    metadata:
+      name: letsencrypt-staging
+    spec:
+      acme:
+        email: certificate-notices@my-company.com
+        privateKeySecretRef:
+          name: letsencrypt-production
+        server: https://acme-staging-v02.api.letsencrypt.org/directory
+        solvers:
+          - http01:
+              ingress:
+                class: contour
+    ```
+
+    > **Important**
+    > * `Let's Encrypts` staging API is not a publicly-trusted CA. You will
+    >   have to add its certificate to your devices trust chain and [TAP's
+    >   custom CA certificates](../custom-ca-certificates.hbs.md).
+    > * `Let's Encrypts` requires your `shared.ingress_domain` to be accessible
+    >   from the internet
+    > * Depending on your setup you will need to adjust 
+    >   [.spec.acme.solvers](https://cert-manager.io/docs/configuration/acme/#solving-challenges)
+    > * Replace `.spec.acme.email` with the email which should receive notices
+    >   for certificates from LetsEncrypt
+
+  : Other
+
+    You can use any other cert-manager-supported `ClusterIssuer` as an ingress issuer for TAP.
+
+    Cert-manager supports a host of in-tree and out-of-tree issuers.
+
+    Refer to cert-manager's [documentaion of issuers](https://cert-manager.io/docs/configuration/).
+
+1. Set `shared.ingress_issuer` to the name of your issuer
+
+  : Custom CA
+
+    ```yaml
+    #! my-tap-values.yaml
+    #! ...
+    shared:
+      ingress_issuer: my-company-ca
+    #! ...
+    ```
+
+  : LetsEncrypt production
+
+    ```yaml
+    #! my-tap-values.yaml
+    #! ...
+    shared:
+      ingress_issuer: letsencrypt-production
+    #! ...
+    ```
+
+  : LetsEncrypt staging
+
+    ```yaml
+    #! my-tap-values.yaml
+    #! ...
+    shared:
+      ingress_issuer: letsencrypt-staging
+    #! ...
+    ```
+
+  : Other
+
+    ```yaml
+    #! my-tap-values.yaml
+    #! ...
+    shared:
+      ingress_issuer: my-company-ca
+    #! ...
+    ```
+
+1. Apply TAP's installation values
+
+Once the configuration is applied, components eventually obtain certificates
```

Comment:
Can you briefly describe how one would conveniently verify the certificates have been updated? It'll be helpful  but I don't think it's absolutely necessary. I find it personally helpful at the end of a procedure to have some hints if I completed it correctly